woocommerce/woocommerce
# FIX strtotime(): Passing null to parameter #1 () of type string is deprecated PHP 8

Fix: https://github.com/woocommerce/woocommerce/issues/40532

## Testing instructions

1. Make sure HPOS is disabled in WC > Settings > Advanced > Features. That is, "WordPress posts storage (legacy)" should be selected as datastore.
2. Make sure you're on PHP > 8.x.
3. Go to page: /wp-admin/post-new.php?post_type=shop_order.
4. With this PR you should **not** have the deprecation message:
   ```
   Deprecated: strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in C:\xampp\htdocs\wordplate\public\plugins\woocommerce\includes\admin\class-wc-admin-post-types.php on line 203
   ```
